### PR TITLE
XLTF7267-BHA+: Document how to change the PLOAM password

### PR DESCRIPTION
**Pull request by nanomad**
Also fix the version string command

### CODE REVIEW

File: _ont_xgs/ont-hisense-ltf7267-bha+.md [hunk 1]
```diff
@@ -307,6 +307,57 @@ INT CFG_ID_PON_VSSN = 0xAABBCCDD;
 
 Reboot ONT to apply the change
 
+## Setting ONU GPON PLOAM password
+{% include alert.html content="The value 0x0 is null, take note of your LoID and password from the original ONT" color="red" %}
```
Comment (simonebortolin) [inline _ont_xgs/ont-hisense-ltf7267-bha+.md:311]: LoID and password ??

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Comment by simonebortolin** [comment]
please also fix

```
In Italy, if you are under some Huawei OLT it’s mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy
```

adding ` to hex values

**Comment by nanomad** [comment]
> please also fix
> 
> ```
> In Italy, if you are under some Huawei OLT it’s mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy
> ```
> 
> adding ` to hex values

Removed since this guidance has no place in the general stick section and it is more of a "compatibility with a specific kind of ISP OLT". Also, even with the updated OMCC version  the PPTP profile requires a manual hack to be working properly

File: _ont_xgs/ont-hisense-ltf7267-bha+.md [hunk 2]
```diff
@@ -307,6 +307,56 @@ INT CFG_ID_PON_VSSN = 0xAABBCCDD;
...
+
+Which is then split into the following blocks (the last block gets padded with 0 to reach 8 digits)
+
+```
+BLOCK 0: 0x41314232
+BLOCK 1: 0x43334434
+BLOCK 2: 0x45350000
+```
+
+Each block is then byte swapped (i.e. with read each block of two digits from right to left)
```
Comment (nanomad) [inline _ont_xgs/ont-hisense-ltf7267-bha+.md:336]:
```suggestion
Each block is then byte swapped (i.e. read each sequence of two digits from right to left)
```

File: _ont_xgs/ont-hisense-ltf7267-bha+.md [hunk 3]
```diff
@@ -307,6 +307,56 @@ INT CFG_ID_PON_VSSN = 0xAABBCCDD;
...
+
+Append lines below to the file and save it to change the PLOAM password
+
+```
+INT             CFG_ID_PON_REGISTRATION_ID0                                         = 0x32423141;
+INT             CFG_ID_PON_REGISTRATION_ID1                                         = 0x34443343;
+INT             CFG_ID_PON_REGISTRATION_ID2                                         = 0x00003545;
+```
+
+Reboot ONT to apply the change.
```
Comment (nanomad) [inline _ont_xgs/ont-hisense-ltf7267-bha+.md:358]:
```suggestion
Reboot the ONT to apply the change.
```

File: _ont_xgs/ont-hisense-ltf7267-bha+.md [hunk 4]
```diff
@@ -307,6 +307,56 @@ INT CFG_ID_PON_VSSN = 0xAABBCCDD;
...
+
+Each block is then byte swapped (i.e. read each sequence of two digits from right to left)
+
+```
+BLOCK 0: 0x32423141
+BLOCK 1: 0x34443343
+BLOCK 2: 0x00003545
+```
+
+And then you can finally save it by chaning the configuration file
```
Comment (nanomad) [inline _ont_xgs/ont-hisense-ltf7267-bha+.md:344]:
```suggestion
And then you can finally persist it by changing the configuration file
```

**Review by simonebortolin — APPROVED** [review]

**Pull request merged**